metadata_backend/helpers/schemas/ena_policy.json

```diff
@@ -282,29 +290,30 @@
         }
     },
     "type": "object",
-    "description": "Describes an object that contains data access policy information.",
+    "description": "Describes an object that contains data access Policy information.",
     "required": [
         "dacRef",
         "policy"
     ],
     "properties": {
         "title": {
             "title": "Policy Title",
-            "description": "Short text that can be used to call out data access policies in searches or in displays.",
+            "description": "Title of the Policy so it can be easily be found.",
             "type": "string"
         },
         "dacRef": {
             "title": "Data Access Committee Reference",
-            "description": "Identifies the data access committee to which this policy pertains.",
+            "description": "Identifies the data access committee to which this Policy pertains.",
             "$ref": "#/definitions/reference"
         },
         "policy": {
             "title": "Policy",
-            "description": "Identifies the data access committee to which this policy pertains.",
+            "description": "Policies can be added either by adding the text of the Policy of by pointing to an existing URL.",
```

Comment:
...text of the Policy `or` by pointing to an ...

Reply:
good catch, fixed.